tests/codeception-setup.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
@javigomez Hey Javi, before the PR against core we move it into the builds folder anyways - we did that for the unit and javascript tests there.